### PR TITLE
BP-9: publish packages to CloudSmith

### PR DESCRIPTION
**Pull request by RangerRick**
This PR adds package-signing to the build, and publishes the packages to CloudSmith.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: 2.1
 executors:
   build-executor:
     docker:
-      - image: opennms/build-env:1.0-b5
+      - image: opennms/build-env:1.8.0.252.b09-3.6.3-b5231
```
Comment (indigo423) [inline .circleci/config.yml:6]: 👍

File: .circleci/config.yml [hunk 2]
```diff
@@ -288,6 +324,12 @@ jobs:
       - store_test_results:
           path: ~/junit
 
+      - sign-packages/sign-rpms:
+          skip_if_forked_pr: true
+          gnupg_home: ~/tmp/gpg
+          gnupg_key: opennms@opennms.org
+          packages: assembly/*-rpm/target/rpm/*-alec-plugin/RPMS/noarch/*.rpm
+
```
Comment (indigo423) [inline .circleci/config.yml:332]: Signed packages :)

**Review by indigo423 — APPROVED** [review]
LGTM! 👍 We keep the publishing jobs to package-cloud. Is this still required?

**Comment by RangerRick** [comment]
> LGTM! 👍 We keep the publishing jobs to package-cloud. Is this still required?

I didn't want to break that in case there was infrastructure still pointing to it somewhere.  Figured I'd leave "getting rid of packagecloud" a separate issue.

**Pull request merged**